src/natcap/invest/datastack.py

```diff
@@ -30,11 +30,6 @@
 import ast
 import warnings
 
-try:
-    basestring
-except NameError:
-    basestring = str
-
 from osgeo import gdal
 from osgeo import ogr
 import six
```

Comment:
`six` is also a python 2/3 compatibility layer, and I'm sorry I didn't notice this before!  Could you also remove a uses of `six` here in favor of their python3 equivalent?  I suspect it'll just be a matter of an iterator or a string type.